x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/fleet/agent_list_page/index.tsx

```diff
@@ -162,30 +281,48 @@ export const AgentListPage: React.FunctionComponent<{}> = () => {
         return null;
       },
     },
+    {
+      field: 'active',
+      name: i18n.translate('xpack.ingestManager.agentList.statusColumnTitle', {
+        defaultMessage: 'Status',
+      }),
+      truncateText: true,
+      render: (active: boolean, agent: any) => <AgentHealth agent={agent} />,
+    },
     {
       field: 'config_id',
       name: i18n.translate('xpack.ingestManager.agentList.configColumnTitle', {
-        defaultMessage: 'Agent Config',
+        defaultMessage: 'Configuration',
       }),
       truncateText: true,
       render: (configId: string) => {
         const configName = agentConfigs.find(p => p.id === configId)?.name;
         return configName ? (
           <ConnectedLink color="primary" path={`/configs/${configId}`}>
-            {configName}
+            {configName}{' '}
+            <EuiTextColor color="subdued">{/* TODO Fix with revision */}v1</EuiTextColor>
           </ConnectedLink>
         ) : (
           <EuiTextColor color="subdued">{configId}</EuiTextColor>
         );
       },
     },
     {
-      field: 'active',
-      name: i18n.translate('xpack.ingestManager.agentList.statusColumnTitle', {
-        defaultMessage: 'Status',
+      field: 'local_metadata.agent_version',
+      name: i18n.translate('xpack.ingestManager.agentList.versionTitle', {
+        defaultMessage: 'Version',
       }),
       truncateText: true,
-      render: (active: boolean, agent: any) => <AgentHealth agent={agent} />,
+      render: (version: string, agent: any) => <EuiText>{version}</EuiText>,
```

Comment:
Do you need a `render()` function here? (It seems to just return the value wrapped in EuiText).